Tools/clinic/clinic.py

```diff
@@ -2733,7 +2733,7 @@ def __init__(self,
                 if isinstance(self.default_type, type):
                     types_str = self.default_type.__name__
                 else:
-                    types_str = ', '.join((cls.__name__ for cls in self.default_type))
+                    types_str = ', '.join(cls.__name__ for cls in self.default_type)
```

Comment:
Let's make it less than 79 chars as well:

```suggestion
                    names = (cls.__name__ for cls in self.default_type)
                    types_str = ', '.join(names)
```

... or maybe:

```suggestion
                    types_str = ', '.join(
                        cls.__name__ for cls in self.default_type
                    )
```